bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/TypeTags.java

```diff
@@ -55,7 +55,8 @@
     public static final int ANYDATA_TAG = NEVER_TAG + 1;
     public static final int RECORD_TYPE_TAG = ANYDATA_TAG + 1;
     public static final int TYPEDESC_TAG = RECORD_TYPE_TAG + 1;
-    public static final int STREAM_TAG = TYPEDESC_TAG + 1;
+    public static final int TYPEREFDESC = TYPEDESC_TAG + 1;
```

Comment:
Do we use this new tag and type constant in runtime?

Reply:
Removed in https://github.com/ballerina-platform/ballerina-lang/pull/31845/commits/5f173211b17851be677ed85e6b7e07798bba33f6